pkgs/development/python-modules/pyvips/default.nix

```
@@ -0,0 +1,30 @@
+{ buildPythonPackage, fetchPypi, pytestrunner, pytestCheckHook, glib, vips, cffi
+, pkg-config, pkgconfig, lib }:
+
+buildPythonPackage rec {
+  pname = "pyvips";
+  version = "2.1.12";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0pg0dxhxgi2m7bb5bi5wpx9hgnbi8ws1bz6w2dldbhi52pizghl4";
+  };
+
+  nativeBuildInputs = [ pytestrunner pkgconfig pkg-config ];
+
+  buildInputs = [ glib vips ];
+
+  propagatedBuildInputs = [ cffi ];
+
+  doCheck = false;
```

please add comment stating why tests aren't ran
```suggestion
  # tests are not included in pypi tarball
  doCheck = false;
```